worli.sh

```diff
@@ -291,99 +284,94 @@ mount /dev/$nisk'1' /media/bootpart
 mount /dev/$nisk'2' /media/winpart
 
 echo " "
-echo "copying windows files to the drive, this may take awhile"
-echo " "
-echo "note: 'WARNING: device write-protected, mounted read-only.' is also normal"
+echo "Copying Windows files to the drive, this may take a while..."
 echo " "
+echo "*NOTE: 'WARNING: Device write-protected, mounted read-only' is also normal*"
 
 mount $iso /tmp/isomount
-
 cp -r /tmp/isomount/boot /media/bootpart
 cp -r /tmp/isomount/efi /media/bootpart
 mkdir /media/bootpart/sources
 cp /tmp/isomount/sources/boot.wim /media/bootpart/sources
 
 if [ -f $wim ]; then
     cp $wim /media/winpart
-else 
+else
     cp /tmp/isomount/sources/install.wim /media/winpart
 fi
 
 umount /tmp/isomount
 
 echo " "
-echo "copying the drivers to the drive..."
-echo " "
```

Comment:
perhaps the stuff below output stuff?

Reply:
it is, check again with split view, might be more clear

Reply:
no i mean the 'echo " "''
you changed 

echo " "
e
echo " "
command 

to 

echo " "
e
command
echo " "